Stack/Opc.Ua.Core/Schema/ApplicationConfiguration.xsd

```diff
@@ -46,6 +46,7 @@
         </xs:annotation>
       </xs:element>
       <xs:element name="RejectedCertificateStore" type="CertificateStoreIdentifier" minOccurs="0" />
+      <xs:element name="MaxRejectedCertificates" type="xs:int" minOccurs="0" />
```

Comment:
Adding this element here may break older stacks using the newer xsd.

Reply:
I hope not, the elements are just ignored, in the past I didn't experience issues. e.g. when elements are not in the right order it happens alot.